pkg/codegen/testing/test/testdata/output-funcs-edgeorder/python/pulumi_myedgeorder/list_configurations.py

```diff
@@ -85,8 +85,8 @@ def list_configurations(configuration_filters: Optional[Sequence[pulumi.InputTyp
 
 
 @_utilities.lift_output_func(list_configurations)
-def list_configurations_output(configuration_filters: Optional[pulumi.Input[Sequence[pulumi.InputType['ConfigurationFilters']]]] = None,
-                               customer_subscription_details: Optional[pulumi.Input[Optional[pulumi.InputType['CustomerSubscriptionDetails']]]] = None,
+def list_configurations_output(configuration_filters: Optional[pulumi.Input[Sequence[pulumi.InputType['ConfigurationFiltersArgs']]]] = None,
```

Comment:
Isn't this going to break someone? Right now, they could have code like this:

```python
configs = list_configurations_output(configuration_filters=ConfigurationFilters(...))
```

After this change, that will no longer type check because it's expecting `ConfigurationFiltersArgs` but we're passing `ConfigureFilters`, right?

To maintain compatibility, wouldn't we have to type it as `Union[pulumi.InputType['ConfigurationFilters'], pulumi.InputType['ConfigurationFiltersArgs']]`?